src/brief/index.html

```diff
@@ -373,7 +373,7 @@
             <div class="footerrowscolumn1 w-col w-col-3 w-col-medium-3"><a href="../brand/" class="footerblocklink transition" translate>Brand Resources</a><a href="../press/" class="footerblocklink transition" translate>Decred in the Press</a><a href="../recruiting" target="_blank" class="footerblocklink transition" translate>Become a Contributor</a></div>
             <div class="footerrowscolumn1 w-col w-col-2 w-col-medium-2"><a href="https://docs.decred.org/research/overview/" target="_blank" class="footerblocklink transition" translate>Technical Overview</a><a href="/business_brief.pdf" target="_blank" class="footerblocklink transition" translate>Business Brief</a><a href="https://blog.decred.org/2018/02/28/2018-Decred-Roadmap/" target="_blank" class="footerblocklink transition" translate>2018 Roadmap</a></div>
             <div class="footerblockstats footerrowscolumn1 w-clearfix w-col w-col-2 w-col-medium-2">
-              <div class="footerblockstatsrow w-clearfix"><a id="footerRelease" class="backgroundturquoise borderradius4 dropshadow footerblockstatslink" href="https://github.com/decred/decred-release/releases/tag/v1.1.2" target="_blank">v1.1.2</a>
+              <div class="footerblockstatsrow w-clearfix"><a id="footerRelease" class="backgroundturquoise borderradius4 dropshadow footerblockstatslink" href="https://github.com/decred/decred-release/releases/tag/v1.3.0" target="_blank">v1.1.2</a>
```

Comment:
The link is updated, but the text is still 1.1.2.  Please update it before the merge.